.github/workflows/cd.yml

```diff
@@ -337,8 +337,8 @@ jobs:
           rm -f versions.json
           echo ${{steps.version-change.outputs.CHANGED}} > changed.txt
           npx lerna ls -all --json > versions.json
-          mv ./packages/vscode-extension/*.vsix .
-          mv ./packages/server/lib/*.exe server.exe
+          test -f ./packages/vscode-extension/*.vsix && mv ./packages/vscode-extension/*.vsix .
```

Comment:
if *.vsix doesn't exist, will the command `test -f` be failed?

Reply:
No error or exception. already have test on VM